### PR TITLE
Fix/exclude nodes

### PR DESCRIPTION
**Pull request by Jasper-Ben**
PR migrated from hellofresh/eks-rolling-update for @jgournet. Would you be so kind to take a look at it please @martin31821?

### CODE REVIEW

**Comment by martin31821** [comment]
lgtm, so far.

**Comment by Jasper-Ben** [comment]
Great, I'll see if I manage to get the README update and GitHub actions working today, so that we can get this tested 👍

**Comment by Jasper-Ben** [comment]
:wave: @jgournet, we have published the PR change as container image at ghcr.io/deinstapel/eks-rolling-update:pr-1.
Would you be so kind to test if this feature works as you expect? Thanks! :slightly_smiling_face:

**Comment by jgournet** [comment]
@Jasper-Ben : firstly: THANK YOU !
Then: I did a quick try, and it looks good - this will be very helpful for us

**Comment by Jasper-Ben** [comment]
> @Jasper-Ben : firstly: THANK YOU ! Then: I did a quick try, and it looks good - this will be very helpful for us

Great, I will merge this now :)

**Comment by jgournet** [comment]
Thanks again !
Just curious: are you planning on publishing images as "latest" ?
(If yes, I'll wait for it - if not, I'll setup a build&push to my own private repo)

**Comment by Jasper-Ben** [comment]
Hi @jgournet :)

The "edge" tag already exists and corresponds to the latest commit on the trunk branch, which hasn't necessarily been included in a release yet. The "latest" tag will be reserved for the latest tagged release (I think we will start tagging releases once we got the pre-existing tests fixed).

Additionally, any commit on the trunk branch are tagged with the commit SHA, so pinning the container to a specific commit on the trunk branch is also possible.

Lastly, PR contributions deemed "safe enough" by us will be built and tagged as "PR-x", with "x" being the PR ID (as the one you used for testing).

See here for currently available tags:
 https://github.com/orgs/deinstapel/packages/container/package/eks-rolling-update

So basically the TL;DR:

Feel free to use the edge tag for now, once we start doing releases, I recommend switching to "latest" for a potentially more stable experience.

As a side note, explicit tags are often the better choice over floating tags such as "latest", unless you know exactly what you are doing. :)
